ui/src/forms/URIInput.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <b-input :value="textValue" @input="onUpdate" />
+</template>
+
+<script lang="ts">
+import { Prop, Component, Vue } from 'vue-property-decorator'
+import { NamedNode } from 'rdf-js'
+import $rdf from '@rdfjs/data-model'
```

Comment:
```suggestion
import * as $rdf from '@rdf-esm/data-model'
```

Reply:
Ok, can you please explain this to me? So that I don't make the mistake again ;)

Reply:
`@rdf-esm/(.+)` are web-optimized forks of their `@rdfjs/$1` counterparts